### DISCUSSION TITLE
[mobile] Errors trying to edit file of pull request

### DISCUSSION THREAD

**Discussion by democat3457** [community/community#39538] [Mobile]
When clicking edit file on a file of a pull request, the first error pops up at the bottom with "An unknown error occurred." After editing the file and trying to commit the changes, the error "Branch does not exist" pops up after trying to press Commit.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.89.0 (83472182)
Device: iPhone 12 mini (iOS 16.0.2)
</details>

**Comment by jfullstackdev** [comment]
as I'm seeing, the mobile version has this bugs

as for now, your workaround : log in a the browser of your mobile, preferably Chrome and there start working
